packages/plans-grid-next/src/types.ts

```diff
@@ -138,35 +134,31 @@ export interface ComparisonGridProps extends CommonGridProps {
 	selectedPlan?: string;
 }
 
+export type ActionCallbackOptions = {
+	cartItemForPlan?: { product_slug: string } | null;
+	currentPlan?: boolean;
```

Comment:
Didn't get too far into these changes today, but thanks for the patience -- I think I'll have a PR up for more thorough review by eod tomorrow

Reply:
> This sounds like something we can derive at the consuming end, so no need to extend the interface here to accept it as a param. WDYT? 🤔

Ah so if I understand correctly, our goal is to simplify the interface more 🤔 I _do_ wholeheartedly agree with this idea. Hoping to confirm what this looks like in practice, because I think I'm missing a step here.

`currentPlan` identifies a spotlight plan, and dictates what kind of function is invoked for the CTA ( manageAddOn / managePlan or upgrade cart items  ) see [here]( https://github.com/Automattic/wp-calypso/pull/88694/files#diff-414f2908c4fdd943075d97bbf756af6a4cc2a5b0faf5fdd045108cb0cb4b4974R193-R195 )

From what i see about deriving at the consuming end, doing so seems like it would muddy responsibilities of our `getActionCallback` function ( logic for determining which CTA to invoke would now exist in the consuming end _and_ the executing end ) Curious if you could elaborate a bit more

Reply:
https://github.com/Automattic/wp-calypso/pull/88694/commits/f881e55d4a1f2f64b7c0553fad43f12bee0c5337

Reply:
Hey @jeyip I left a couple of comments on the commit that you linked to concerning `currentPlan`. I don't see them - let me know if they came through or if I should repost on the main changes instead. :D

Reply:
Hey thanks for the heads up. I saw them _and_ responded to them, and then subsequently lost them again 😆

Finally found them for a second time, so here's a link for posterity https://github.com/Automattic/wp-calypso/commit/f881e55d4a1f2f64b7c0553fad43f12bee0c5337#r140558754